components/PostCard.tsx

```diff
@@ -28,9 +58,16 @@ const toBase64 = (str: string) => Buffer.from(str).toString("base64");
 const PostCard = ({ post }: Props) => {
   const dispatch = useAppDispatch();
 
+  const isFuturePost = today.toISOString() > post.date ? false : true;
```

Comment:
What's that?

Why do you duplicate helper function?

`helpers/checkOfDraftOrFuturePost.ts`